ext/opcache/jit/zend_jit.c

```diff
@@ -2647,6 +2647,31 @@ static bool zend_jit_next_is_send_result(const zend_op *opline)
 	return 0;
 }
 
+static bool zend_can_jit_compound_binary_op(zend_uchar op, uint32_t op1_info, uint32_t op2_info)
```

Comment:
The name is really bad. "compound" should be removed. "can" should be replaced to some better word.
e.g. zend_jit_profitable_binary_op() or zend_jit_supported_binary_op() or something better.